lib/gds_api/rummager.rb

```diff
@@ -4,17 +4,32 @@
 module GdsApi
   class Rummager < Base
 
+    # Unified search
+    #
```

Comment:
The names "Unified search" and "Advanced search" are outdated now. We changed all the internal code to just call it search and prefixed all the old stuff with "legacy" so its obvious its deprecated. Maybe we should alias the endpoints as well so we have a common language. (We shouldn't remove or rename anything without versioning the API though).